Tryouts/Core/BasicModels/Modules/ProcessesContainer.cs

```diff
@@ -0,0 +1,151 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using MorganStanley.ComposeUI.Tryouts.Core.Abstractions.Modules;
+using NP.Concepts.Behaviors;
+using System.Collections.ObjectModel;
+
+namespace MorganStanley.ComposeUI.Tryouts.Core.BasicModels.Modules
+{
+    public class ProcessesContainer
```

Comment:
Where/how is this class used? I don't see any references to it.

Reply:
very good point - it was one of the working sketches that got checked in by mistake.

Reply:
Removed the unneeded file